### DISCUSSION TITLE
Build errors

### DISCUSSION THREAD

**Discussion by JustinKerkettaLeidos** [zodiacon/ObjectExplorer#1] [General]
I have added the path to WTL to the projects ObjExp and WTLHelper but get the following errors. Please help if you can. I am trying to follow along with Pavel Y.'s course on Kernel Mechanisms on PluralSight and being able to run this tool would help enhance one's learning a lot.

![image](https://github.com/zodiacon/ObjectExplorer/assets/129797288/1e97c399-24af-4a51-b0b9-50554bf0b311)

![image](https://github.com/zodiacon/ObjectExplorer/assets/129797288/86c20fad-0a42-46a4-9672-c9b8b864c35b)

Cannot open include file: 'detours/detours.h': No such file or directory	WTLHelper	C:\Temp\ObjectExplorer\WTLHelper\WTLHelper\ThemeHelper.cpp	3
	
Error	C1083	Cannot open include file: 'wil\resource.h': No such file or directory	WTLHelper	C:\Temp\ObjectExplorer\WTLHelper\WTLHelper\ListViewhelper.cpp	4
	
Error (active)	E1696	cannot open source file "wil\resource.h"	ObjExp	C:\Temp\ObjectExplorer\ObjExp\pch.h	42

**Comment by zodiacon** [comment]
I switched to using vcpkg.
Install vcpkg, and then 
vcpkg integrate install
vcpkg install wil:x64-windows
vcpkg install detours:x64-windows
vcpkg install wtl:x64-windows

**Reply by JustinKerkettaLeidos** [reply]
Thanks @zodiacon, especially since your reply was prompt. That worked. 
Following your course in conjunction with your book.

![image](https://github.com/zodiacon/ObjectExplorer/assets/129797288/c81ba0fd-8aa6-45ee-9779-4001ef4d5e7d)
